components/ServerCard.tsx

```diff
@@ -1,28 +1,13 @@
 import Image from "next/image"
 import { FormattedMessage } from "react-intl"
 import LinkButton from "./LinkButton"
+import type Server from "../types/server"
 
-let server = {
-  domain: "mastodon.social",
-  version: "3.5.3",
-  description:
-    "The original server maintained by the Mastodon gGmbH non-profit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam lorem tellus, dignissim sed faucibus in, auctor ut mi.",
-  languages: ["en"],
-  region: "",
-  categories: ["general"],
-  proxied_thumbnail:
-    "https://proxy.joinmastodon.org/d7d02f9615184131475feeb95ab8cd01e6575448/68747470733a2f2f66696c65732e6d6173746f646f6e2e736f6369616c2f736974655f75706c6f6164732f66696c65732f3030302f3030302f3030312f6f726967696e616c2f766c63736e61702d323031382d30382d32372d31366834336d3131733132372e706e67",
-  total_users: 725761,
-  last_week_users: 43440,
-  approval_required: true,
-  language: "en",
-  category: "general",
-}
-
-const ServerCard = () => {
+const ServerCard = ({ server }: { server: Server }) => {
+  let category = server.category === "lgbt" ? "LGBTQ+" : server.category
```

Comment:
not sure if this is placeholder logic, and doesn't need to happen in this PR, but we should copy over the localized category strings https://github.com/oakstudios/joinmastodon/blob/caace73b02dcbf97a1c9c1b5fe4c53e4ad48ca23/z_archive/locales/en.json#L95-L110

Reply:
oh right! I forgot that these are translated, will do that in this PR

Reply:
updated here c996378f366ccf79a3d13a720a1e374dceec4dac